pkgs/star-citizen/default.nix

```diff
@@ -0,0 +1,101 @@
+{
+  lib,
+  makeDesktopItem,
+  symlinkJoin,
+  writeShellScriptBin,
+  gamemode,
+  winetricks,
+  wine,
+  dxvk,
+  wineFlags ? "",
+  pname ? "star-citizen",
+  location ? "$HOME/Games/star-citizen",
+  tricks ? ["arial" "vcrun2019" "win10"],
+  preCommands ? "",
+  postCommands ? "",
+}: let
+  version = "1.6.7";
+  src = builtins.fetchurl rec {
+    url = "https://install.robertsspaceindustries.com/star-citizen/RSI-Setup-${version}.exe";
+    name = "RSI-Setup-${version}.exe";
+    sha256 = "07l0585xxyqczqdyfg3zah1sbavkyl1ybjay6mc9xrxr6mhmm4xm";
+  };
+
+  # concat winetricks args
+  tricksFmt = with builtins;
+    if (length tricks) > 0
+    then concatStringsSep " " tricks
+    else "-V";
+
+  script = writeShellScriptBin pname ''
+    export WINEARCH="win64"
+    export DXVK_HUD=fps,compiler
+    export WINEFSYNC=1
+    export WINEESYNC=1
+    export WINEPREFIX="${location}"
+    # Anti-cheat
+    export SteamGameId="starcitizen"
+    __GL_SHADER_DISK_CACHE=1
+    __GL_SHADER_DISK_CACHE_SIZE=1073741824
+    __GL_THREADED_OPTIMIZATIONS=1
+    PATH=$PATH:${wine}/bin:${winetricks}/bin
```

Comment:
Perhaps we could use `PATH=$PATH:${lib.makeBinPath [wine winetricks]}` here, though it's not strictly necessary.